proteuslib/unit_models/tests/test_reverse_osmosis_0D.py

```diff
@@ -586,10 +587,9 @@ def test_Pdrop_calculation(self):
         feed_flow_mass = 1
         feed_mass_frac_NaCl = 0.035
         feed_mass_frac_H2O = 1 - feed_mass_frac_NaCl
-        feed_pressure = 50e5
+        feed_pressure = 85e5
```

Comment:
Does this test require 85 bar? Does it not solve with 55 bar?

Reply:
It does solve with 50 bar.